common/djangoapps/enrollment/views.py

```diff
@@ -668,6 +669,17 @@ def post(self, request):
             if email_opt_in is not None:
                 org = course_id.org
                 update_email_opt_in(request.user, org, email_opt_in)
+            else:
+                # This code is for REV-19 to handle the opt in flag
+                try:
+                    opt_in = ExperimentData.objects.get(
```

Comment:
This value isn't used... is that intentional? You will also likely need to coerce it to the correct type.